src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,62 @@
+package duke;
+import java.util.Scanner;
+
+/**
+ * This is the User Interface of the Duke. Accepts keyboard input from user,
+ * and shows the output after the operations.
+ */
+public class Ui {
+
+    private Scanner sc;
+    private Parser parser;
+
+    /**
+     * Ui Constructor. Has a Parser attribute to process inputs,
+     * and a Scanner to accept inputs. Shows a logo to greet the user.
+     */
+    public Ui() {
+        this.sc = new Scanner(System.in);
+        this.parser = new Parser();
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+    }
+
+    /**
+     * Starts the UI. The UI will continue to receive inputs, and terminates
+     * when the user wishes to.
+     * @param taskList
+     */
+    public String start(TaskList taskList, String input) {
+        //showLine();
+        String output = "";
+        try {
+            String command = input.trim();
+            output = parser.processCommand(command, taskList);
+            } catch (DukeTerminateException e) {
+                output = "Bye! See you again!";
+            } catch (InvalidInputException e) {
+                output =  e.toString();
+            } catch (DukeException e) {
+                output = e.toString();
+            } finally {
+                return output;
+            }
+    }
```

Comment:
UI is only responsible for displaying some outputs and not handling the main execution of the program. This function should not be here!